test.cfg.example

```diff
@@ -0,0 +1,15 @@
+# To run tests, copy this file to test.cfg and set the values below appropriately.
+
+[idmappingservicetest]
+
+# path to the mongodb executable to use for the tests.
+test.mongo.exe=/path/to/mongodbexecutable
+
+# true to use wired tiger, anything else for false.
+test.mongo.wired_tiger=false
+
+# path to a temporary directory to use for tests.
+test.temp.dir=/path/to/temp/dir/make/sure/this/is/safe
+
+# true to keep the temporary directory contents after running tests. 
+test.temp.dir.keep=false
```

Comment:
does this need to be an option? Maybe you can always keep test data after a test and wipe it before running again

Reply:
The reason it's an option is that sometimes the test data can get really big - the workspace is an example of that. I never wanted that much crap on my system unless I was debugging, and so made this an option. All of my other modules are the same way.